uwblueprint/focus-on-nature
# feat: add delete camp endpoint

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/Task-Board-S22-1a3e651ab8544192bf0fdcbba04f6d03?p=36046d9832f345aa824e498808614fcc)



## Implementation description
* added delete camp endpoint which takes in a camp id and deletes the camp's form questions, file, and camp sessions
* modified camp session deletion to also delete the associated campers



## Steps to test
1. Create a camp with an associated file, form questions, and camp sessions each with campers
2. Ensure deletion will delete all these fields from the database
3. If a camp session deletion fails, the program shouldn't throw an error in the middle of camp sessions deletions, but rather at the end



## What should reviewers focus on?
* Ensure error handling is correct for camp session deletion


## Checklist
